src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -55,12 +53,14 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
```

Comment:
This isn't related to your change @atavism, but it looks like init() in this file is actually never called. I'm unclear on how this is working at all, since that would mean that clientConfig is never created and should be nil.